### ISSUE TITLE
Install fails with latest node version  17

### ISSUE THREAD

**Issue by bprobbins** [LowPowerLab/RaspberryPi-Gateway#43]
But luckily, if I insert `sudo node-install -v 16.12.0` immediately after line 113 (`sudo wget`....) and comment out `sudo npm install -g npm` , it works. Seems the node module serialport doesn't yet play nice with latest nodejs.

**Comment by LowPowerLab** [comment]
@bprobbins - thanks but how does it fail?
I just did a clean install on latest raspberry OS 5'21 and it's all good. Haven't seen any errors. Node v17.0.1, serialport works, data comes in through USB or GPIO serial.

**Comment by bprobbins** [comment]
Yes, I did get data coming thru the Serial but attempts to connect to the server user interface resulted in the message "waiting for socket" I recall. It never went beyond that message. Tries it on a Pi 2 & Pi 3 (newest updates OS) with same result. I did look at the server log which displayed messages that nconf was not found as well as errors associated with serialport. Similar problem messages occurred when I ran node gateway.js in terminal. But on further thought maybe the primary problem was more likely some compatibility issues with nconf. I may have erroneously jumped to  serialport as the problem because I have had similar issues with serialport/node version incompatibility in the past.
When you did the clean install did you try to log into the user interface after install? That's what I should have mentioned was where the fail evidenced itself.
